staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -57,10 +57,10 @@ type Patcher struct {
 	Overwrite bool
 	BackOff   clockwork.Clock
 
-	Force       bool
-	Cascade     bool
-	Timeout     time.Duration
-	GracePeriod int
+	Force          bool
+	DeleteStratedy cmdutil.DeleteStrategy
```

Comment:
What about `CascadingStrategy`? Also, you have a typo here that you "propagated" all over the place

Reply:
Done :) And thanks for pointing out my typo.